### ISSUE TITLE
Forward projection bug related to the voxel spacing

### ISSUE THREAD

**Issue by oooo1114** [LLNL/LEAP#70]
Hi,
I have tested LEAP with my geometry.
When the voxel spacing (width or height) is smaller than the real pixel width or height in the isocenter,  i.e. pixel width or height * sod / sdd, the forward projection goes wrong. Most views are all zeroes, except for tens of weird views near the center of angles.
Here are the scripts which can reproduce the bug:
```
import sys
import os
import time
import numpy as np
from leapctype import *
leapct = tomographicModels()
numCols = 860
numTurns = 6
numAngles = 8640
numRows = 32
pixelHeight = 1.12
pixelWidth = 1.0645
centerRow = 0.5*(numRows-1)
centerCol = 0.5*(numCols-1)-4.2412
phis = leapct.setAngleArray(numAngles, 360.0*numTurns)
sod = 570
sdd = 1005
pitch = 0.861
leapct.set_conebeam(numAngles, numRows, numCols, pixelHeight, pixelWidth, centerRow, centerCol, phis, sod, sdd)
leapct.set_normalizedHelicalPitch(pitch)
leapct.set_curvedDetector()

numX = 512
numY = 512
numZ = 137
voxelWidth = 0.6 # set to 0.6 will trigger the bug
# voxelWidth = 0.7 # set to 0.7 is the normal case
voxelHeight = 1.26
offsetX = 0
offsetY = 0
offsetZ = 0
leapct.set_volume(numX, numY, numZ, voxelWidth, voxelHeight, offsetX, offsetY, offsetZ)
leapct.set_diameterFOV(640)
# Trouble-Shooting Functions
leapct.print_parameters()
# leapct.sketch_system()
g = leapct.allocateProjections()
f = leapct.allocateVolume()
leapct.set_FORBILD(f,True)
leapct.project(g,f)
```
When voxelWidth is set to 0.7, the projection looks fine. See the No. 4161 view below.
![image](https://github.com/LLNL/LEAP/assets/24436961/13441a23-3041-4742-9911-bb329ee4b03b)

When voxelWidth is set to 0.6, the projection looks weird. See the No. 4161 view below.
![image](https://github.com/LLNL/LEAP/assets/24436961/f36e14ea-c879-4f0e-94ac-0e359c1b5390)

**Comment by kylechampley** [comment]
This bug has been fixed and new code merged into the main branch.  We also made a new release with this bug fix, [v1.16](https://github.com/LLNL/LEAP/releases/tag/v1.16).

When fixing this bug we also found that the scaling factor of the helical FBP algorithm was a little off and we fixed this.

Thanks for reporting this bug!

**Comment by oooo1114** [comment]
Thanks for your work!
